### PR TITLE
fix: onBlur gets triggered when selecting date in RangePicker

### LINKED ISSUE #38350: onBlur gets triggered when selecting date in range picker
### Reproduction link

[Edit on CodeSandbox](https://codesandbox.io/s/ant-rangepicker-6gmz85?file=/src/Working.tsx)

### Steps to reproduce

Open the range picker , and try selecting the range

### What is expected?

It should let us select the range

### What is actually happening?

Picker is closing 

| Environment | Info |
| --- | --- |
| antd | undefined |
| React | 17.0 |
| System | Mac OS Monterey |
| Browser | Version 106.0.5249.119 (Official Build) (arm64) |

### PR DESCRIPTION
**Pull request by kiner-tang**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [X] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

- fix: #38350

https://github.com/react-component/picker/pull/510

### 💡 Background and solution

feat(picker): update picker version

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | update DatePicker version  |
| 🇨🇳 Chinese | 更新日期选择框版本 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -139,7 +139,7 @@
     "rc-motion": "^2.6.1",
     "rc-notification": "~4.6.0",
     "rc-pagination": "~3.2.0",
-    "rc-picker": "~2.6.11",
+    "rc-picker": "^2.7.0",
```
Comment (afc163) [inline package.json:142]:
```suggestion
    "rc-picker": "~2.7.0",
```

File: tests/shared/focusTest.tsx [hunk 2]
```diff
@@ -110,7 +110,7 @@ export default function focusTest(
         expect(blurred).toBeTruthy();
 
         fireEvent.blur(getElement(container));
-        await sleep(0);
+        await sleep(110);
```
Comment (afc163) [inline tests/shared/focusTest.tsx:113]: 不应该改这个，其他组件并没有被影响。

Comment (afc163) [reply tests/shared/focusTest.tsx:113]: 这样看上去不太好，RangePicker onBlur 触发时机不应该被改变。

Comment (kiner-tang) [reply tests/shared/focusTest.tsx:113]: 如果是`RangePicker`时，由于存在多个`input`之间焦点切换的情况，是有一些延迟的

Comment (kiner-tang) [reply tests/shared/focusTest.tsx:113]: image

Comment (afc163) [reply tests/shared/focusTest.tsx:113]: 单独延长 RangePicker 的 onBlur 测试用例检测时机吧。

Comment (kiner-tang) [reply tests/shared/focusTest.tsx:113]: 恩恩好的

Comment (kiner-tang) [reply tests/shared/focusTest.tsx]: done

**Review by afc163 — APPROVED** [review]

**Comment by afc163** [comment]
@zombieJ 看看这个方案。

**Comment by kiner-tang** [comment]
> @zombieJ 看看这个方案。

@zombieJ 可以帮忙看一下吗？

**Pull request merged**